package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@keycloak/keycloak-admin-client",
-  "version": "17.0.0-dev.5",
+  "version": "17.0.0-dev.6",
```

Comment:
We should not be bumping this version in PRs, the CI will publish new versions automatically on merge.